src/Installers/Windows/Wix.targets

```diff
@@ -26,15 +26,19 @@
   </Target>
 
   <Target Name="_GenerateBundleGuids" Condition="'$(OutputType)' == 'bundle'">
-    <GenerateGuid NamespaceGuid="$(NamespaceGuid)" Values="$(BundleNameShort);$(GuidInputs)">
+    <!-- Don't need to use the name of the bundle since we have separate namespace GUIDs. The name will also changes based on the milestone and destabalize
+         the upgrade code. Bundle upgrades pivot on Major.Minor.Patch changes. For example, 3.0.1-preview 1 can upgrade to 3.0.1-preview 8, but 3.0.1
+         cannot upgrade to 3.0.2 or 3.1. -->
+    <BundleGuidInputs>$(AspNetCoreMajorVersion).$(AspNetCoreMinorVersion).$(AspNetCorePatchVersion);$(Platform)</BundleGuidInputs>
```

Comment:
Consider using an arbitrary string constant in 'BundleGuidInputs'. That was our answer to the possibility that a product could exist in the world that seeded a GUID generation with: "3.0.0;x64"?

Reply:
That's why we have a namespace GUID. With UUID3, the same inputs generate the same GUID within a namespace. We have one namespace for ASP.NET Core runtime and one for the hosting bundle, so the GUID inputs are effectively something like "AspNetRuntime;3.0.0;x86" and "Hosting;3.0.0;x86" ensuring that those two don't clash